.vimrc

```diff
@@ -56,11 +56,10 @@ Plug 'junegunn/vim-easy-align'         " Easy Align
                                            " Example: gaip-=        -> go align inner paragraph around the last =
                                            " Example: gaip*=        -> go align inner paragraph around all =
                                            " Example: gaip<Enter>*= -> go align inner paragraph, reversed, around all =
-Plug 'leafgarland/typescript-vim'      " Typescript support
```

Comment:
Removal is not part of adding coc.nvim, strictly speaking. But this should still be removed-- I haven't been using it.